README.md

```diff
@@ -37,45 +37,190 @@ However, processing WSIs is far from being trivial. First of all, WSIs can be st
 
 The aim of this project is to provide a tool for WSI processing in a reproducible environment to support clinical and scientific research. HistoLab is designed to handle WSIs, automatically detect the tissue, and retrieve informative tiles, and it can thus be integrated in a deep learning pipeline.
 
-## Getting Started
+## Getting Started 
 
-### Prerequisites
+### Prerequisites 
 
 HistoLab has only one sistem-wide dependency: OpenSlide.
 
 You can download and install it from [OpenSlide](https://openslide.org/download/) according to your operating system.
 
 
-### Installation
+### Installation 
 
 ```
 pip install histolab
 ```
 
-### Quickstart
+### Quickstart 
 
-COMING SOON!
+```python
+from histolab.data import breast_tissue, heart_tissue
+```
+
+
+```python
+from PIL import Image
+from IPython.display import Image as show_image
+```
+
+The module ```data``` contains some WSIs retrieved from the [TCGA data portal](https://portal.gdc.cancer.gov/) and [OpenSlide test data](http://openslide.cs.cmu.edu/download/openslide-testdata/Aperio/), to be used as examples. The available images are H&E tissue sections stored in .svs format:
+* ```aorta_tissue```: Aorta tissue slide (OpenSlide)
```

Comment:
for the quick start it's not really needed to have all of these explained. Let's keep them for a longer tutorial

Reply:
OK